src/django/api/templates/mail/facility_list_complete_body.txt

```diff
@@ -44,5 +44,5 @@ For any questions, feel free to reach out to the OS Hub team: info@opensupplyhub
 
```

Comment:
On line 43 we include the info@ email in the body text. I would suggest changing it to data@ to harmonize with the from address.

Reply:
https://github.com/open-apparel-registry/open-apparel-registry/pull/2373/commits/9ddf0633170f9e9ca9b01a61bb935e6be3636ef0